lib/rules/property-no-vendor-prefix/__tests__/index.js

```diff
@@ -100,3 +100,46 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [
+    true,
+    { ignoreProperties: ["transform", "columns", "/^animation-/"] }
+  ],
+
+  accept: [
+    {
+      code: "a { -webkit-transform: scale(1); }"
+    },
+    {
+      code: "a { -moz-columns: 2; }"
+    },
+    {
+      code: ".foo { -webkit-animation-delay: 0.5s; }"
+    },
+    {
+      code: ".foo { width: 320px; }"
+    }
+  ],
+  reject: [
+    {
+      code: "-webkit-border-radius: 10px;",
```

Comment:
Replace with `a { -webkit-border-radius: 10px; }`, because property outside a rule isn't a valid CSS.

Reply:
@nalinbhardwaj  FYI, this requirement is in the [guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#example-patterns).

Reply:
Fixing. There are some more tests in the repo that violate this, [example](https://github.com/stylelint/stylelint/blob/master/lib/rules/value-no-vendor-prefix/__tests__/index.js#L137)